zebra-state/src/service/finalized_state/disk_format/upgrade.rs

```diff
@@ -74,6 +78,31 @@ pub struct DbFormatChangeThreadHandle {
     should_cancel_format_change: Arc<AtomicBool>,
 }
 
+/// Defines method signature for running disk format upgrades.
+pub trait DiskFormatUpgrade {
+    /// Runs disk format upgrade.
+    fn run(
+        &self,
+        initial_tip_height: Height,
+        network: Network,
+        upgrade_db: &ZebraDb,
+        should_cancel_format_change: Arc<AtomicBool>,
+    );
+}
+
+fn format_upgrades() -> Vec<(Version, Box<dyn DiskFormatUpgrade>)> {
+    fn parse_expect_version(version: &str) -> Version {
+        version
+            .parse()
+            .expect("Hardcoded version string should be valid.")
+    }
+
+    vec![
+        (parse_expect_version("25.1.0"), Box::new(PruneTrees)),
+        (parse_expect_version("25.2.0"), Box::new(AddSubtrees)),
```

Comment:
Since the `25.2.0` version change is going to be merged in a previous PR, and merging that PR will run CI, we can't re-use that version in this PR. Instead, finish any changes to the format in this PR, and then increment the version again.

Normally we could delete all of the `25.2` cached states in Google Cloud, but I don't think that will work here, because it's a manual step that we can't make Mergify do. (And if we don't do that manual step before each CI run, the upgrade code in this PR will be skipped during the tests.)

```suggestion
        (parse_expect_version("25.3.0"), Box::new(AddSubtrees)),
```

Reply:
> Since the 25.2.0 version change is going to be merged in a previous PR, and merging that PR will run CI, we can't re-use that version in this PR. Instead, finish any changes to the format in this PR, and then increment the version again.

The previous PR shouldn't merge into main without this first merging into it.

Reply:
Ah, then we'll need to delete all the `25.2` states before doing the final tests and merging both PRs.

In fact, we might need to do that deletion on a regular basis, to get decent CI results from both PRs.